atlasdb-client/src/test/java/com/palantir/atlasdb/logging/SafeLoggableDataUtilsTest.java

```diff
@@ -0,0 +1,141 @@
+import com.palantir.atlasdb.table.description.NameComponentDescription;
+import com.palantir.atlasdb.table.description.NameMetadataDescription;
+import com.palantir.atlasdb.table.description.NamedColumnDescription;
+import com.palantir.atlasdb.table.description.TableMetadata;
+import com.palantir.atlasdb.table.description.UniformRowNamePartitioner;
+import com.palantir.atlasdb.table.description.ValueType;
+import com.palantir.atlasdb.transaction.api.ConflictHandler;
+
+public class SafeLoggableDataUtilsTest {
+    private static final TableReference TABLE_REFERENCE_1 = TableReference.createFromFullyQualifiedName("atlas.db");
+    private static final TableReference TABLE_REFERENCE_2 = TableReference.createFromFullyQualifiedName("atlas.db2");
+    private static final TableReference TABLE_REFERENCE_3 = TableReference.createFromFullyQualifiedName("atlas.db3");
+
+    private static final String ROW_COMPONENT_NAME = "foo";
+    private static final String COLUMN_LONG_NAME = "barrrr";
+    private static final NameMetadataDescription NAME_METADATA_DESCRIPTION = NameMetadataDescription.create(
+            ImmutableList.of(
+                    new NameComponentDescription(
+                            ROW_COMPONENT_NAME,
+                            ValueType.VAR_LONG,
+                            TableMetadataPersistence.ValueByteOrder.ASCENDING,
+                            new UniformRowNamePartitioner(ValueType.VAR_LONG),
+                            null,
+                            true)),
+            false);
+    private static final ColumnMetadataDescription COLUMN_METADATA_DESCRIPTION =
+            new ColumnMetadataDescription(ImmutableList.of(new NamedColumnDescription("bar", COLUMN_LONG_NAME,
+                    ColumnValueDescription.forType(ValueType.VAR_LONG), true)));
+
+    private static final TableMetadata TABLE_METADATA_1 = new TableMetadata(
+            NameMetadataDescription.create(
+                    ImmutableList.of(new NameComponentDescription(ROW_COMPONENT_NAME, ValueType.VAR_LONG)), false),
+            new ColumnMetadataDescription(
+                    ImmutableList.of(new NamedColumnDescription("bar", "barrrr",
+                            ColumnValueDescription.forType(ValueType.VAR_LONG)))),
+            ConflictHandler.RETRY_ON_WRITE_WRITE);
+    private static final TableMetadata TABLE_METADATA_2 = createTableMetadataWithLoggingAllowed(
+            NAME_METADATA_DESCRIPTION, COLUMN_METADATA_DESCRIPTION);
+    private static final TableMetadata TABLE_METADATA_DYNAMIC_COLUMNS = createTableMetadataWithLoggingAllowed(
+            NAME_METADATA_DESCRIPTION,
+            new ColumnMetadataDescription()
+    );
+
+    @Test
+    public void nothingLoggableByDefault() {
+        ImmutableSafeLoggableData.Builder builder = ImmutableSafeLoggableData.builder();
+        SafeLoggableDataUtils.addLoggableNamesToBuilder(builder, TABLE_REFERENCE_1, TABLE_METADATA_1);
```

Comment:
is it actually necessary to use this method here? seems like we could pass a map into the public constructor